pkg/daemon/ceph/client/command.go

```diff
@@ -166,7 +166,7 @@ func (c *CephToolCommand) run() ([]byte, error) {
 	if command == RBDTool {
 		if c.RemoteExecution {
 			output, stderr, err = c.context.RemoteExecutor.ExecCommandInContainerWithFullOutputWithTimeout(ProxyAppLabel, CommandProxyInitContainerName, c.clusterInfo.Namespace, append([]string{command}, args...)...)
-			if stderr != "" || err != nil {
```

Comment:
How about changing to && instead of removing the check for stderr? If it's empty, we would be appending a `.` for no reason.

Reply:
Using just `err != nil` per huddle discussion.